internal/cmd/gocdk/biome.go

```diff
@@ -36,16 +36,21 @@ type biomeConfig struct {
 	Launcher     *string `json:"launcher,omitempty"`
 }
 
-// findBiomeDir returns the path to the named biome.
-func findBiomeDir(moduleRoot, name string) string {
-	return filepath.Join(moduleRoot, "biomes", name)
+// biomeRootDir returns the path to the biomes directory.
+func biomeRootDir(moduleRoot string) string {
```

Comment:
The root directory is named `biomes` rather than `biome`, should we make that match here?

Reply:
Sure, done.